src/firebolt/async_db/cursor.py

```diff
@@ -328,17 +331,35 @@ def _append_row_set(
 
     async def _api_request(
         self,
-        query: str,
-        set_parameters: Optional[dict] = None,
+        query: Optional[str] = "",
+        parameters: Optional[dict] = None,
+        url_suffix: Optional[str] = "",
     ) -> Response:
+        """
+        Query API, return Response object.
+
+        Args:
+            query (str): SQL query
+            parameters (Optional[Sequence[ParameterType]]): A sequence of substitution
+                parameters. Used to replace '?' placeholders inside a query with
+                actual values. Note: In order to "output_format" dict value, it
+                    must be an empty string. If no value not specified,
+                    JSON_OUTPUT_FORMAT will be used.
+            url_suffix (str): endpoint suffix, for example "cancel" or "status"
+        """
+        if not parameters:
+            parameters = {"output_format": JSON_OUTPUT_FORMAT}
+        elif "output_format" not in parameters:
+            parameters["output_format"] = JSON_OUTPUT_FORMAT
+        elif not parameters["output_format"]:  # if "output_format" is empty string
+            del parameters["output_format"]
```

Comment:
When would this happen?

Reply:
In order to make sure that `output_format` isn't always set to `JSON_OUTPUT_FORMAT` we may occasionally need to send in an empty value. In that case, it needs to be removed or our API endpoint string will be malformed.

Reply:
That is kind of a weird functionality and confusing. Alternately, I could add a parameter to the function signature for the `output_format` value.

Reply:
See further discussion: https://github.com/firebolt-db/firebolt-python-sdk/pull/192#discussion_r941045329

Reply:
This was dealt with. The user does not have the option to choose output formats.